app/services/tts_service.rb

```diff
@@ -54,14 +70,15 @@ def call
 
 private
 
-  def init_request_data(text, format)
+  def init_request_data(text, format, duration, voice)
     # The request data format as required by the currently used TTS-service.
     {"Engine": "standard",
      "LanguageCode": "is-IS",
      "OutputFormat": "mp3",
+     "Duration": "#{duration}",
      "SampleRate": "22050",
-     "TextType": "#{format}",
-     "VoiceId": "Alfur",
+     "TextFormat": "#{format}",
```

Comment:
Has this changed inside the API interface ?

Reply:
yes, to prevent a name clash with the ssml-text-type

Reply:
ok, but then we are not compatible anymore with the original web service and we need to change also the OpenAPI spec